### DISCUSSION TITLE
Take as needed option

### DISCUSSION THREAD

**Discussion by SeventhUnderhand** [Futsch1/medTimer#193] [Ideas]
Is it possible to do a take as needed option that has specified dosage just as scheduled doses does? I take a medication whenever it is needed per 24/hrs and it would be nice to be able to track that.

**Comment by Futsch1** [comment]
The button "Log additional dose" on the overview screen does exactly this: it logs a dose at the current time of either a custom medicine or one of the configured ones. The only difference is that the amount is always indicated manually.

Would this solve your use case or does your idea go beyond that?

**Comment by SeventhUnderhand** [comment]
I did notice this after I sent this but it would be nice to have specified doses instead of just manual.

-------- Original Message --------

**Comment by Futsch1** [comment]
I will implement it, see #215 and #214 . Would this fit your use case?

**Comment by SeventhUnderhand** [comment]
Yes

-------- Original Message --------

**Comment by Futsch1** [comment]
Implemented in v1.9.0, will soon land on Google Play, F-Droid and Izzy
